Updates to query parameters documentation

susodapop opened the pull request:
Straightforward edits. Added a warning about SQL injection. I'm not certain if the documentation is accurate when it says that use of _any query parameter_ requires Full Access to the database (see line 49). I'd like to remove that if it isn't true.

arikfr commented on website/_kb/user-guide/querying/query-parameters.md:
```diff
@@ -10,46 +10,68 @@ title: Query Parameters
 
+### Date Parameters
+
+In many DBAPI's, you can use a string value to represent a date/time which is equivalent to a Text parameter in Redash. Redash's Date parameters are preferable because they have a special calendar interface to make choosing a date easy. If you know in advance, however, that your date parameters will always come from a short list, you can instead use the `Dropdown List` or `Query Based Dropdown Li ...
```
"In many DBAPI's" -> "With many databases,"

susodapop commented:
Updated based on your feedback. I added my first PNG to the docs -- it didn't render properly on my development server but I think it should work on Netlify.

arikfr commented:
> I added my first PNG to the docs -- it didn't render properly on my development server but I think it should work on Netlify.

In what way it didn't render properly?

arikfr commented:
Looks good 👍 

Let's remove the unrelated change in `getting-started.md` and merge this?
